esp/src/src-react/components/GetProtectedList.tsx

```diff
@@ -0,0 +1,80 @@
+import * as React from "react";
+import { DefaultButton, DetailsList, DetailsListLayoutMode, IColumn } from "@fluentui/react";
+import { SizeMe } from "react-sizeme";
+import { csvParse } from "d3-dsv";
+import { DaliService, WsDali } from "@hpcc-js/comms";
+import { scopedLogger } from "@hpcc-js/util";
+import { TableGroup } from "./forms/Groups";
+import nlsHPCC from "src/nlsHPCC";
+import { HolyGrail } from "../layouts/HolyGrail";
+
+class MyDaliService extends DaliService {
+    GetProtectedList(request: WsDali.GetProtectedListRequest): Promise<WsDali.ResultResponse> {
+        return this._connection.send("GetProtectedList", request, "json", false, undefined, "ResultResponse");
+    }
+}
+
+const logger = scopedLogger("src-react/components/GetProtectedList.tsx");
+
+const myDaliService = new MyDaliService({ baseUrl: "" });
+
+interface GetProtectedListProps {
+}
+
+export const GetProtectedList: React.FunctionComponent<GetProtectedListProps> = ({
+
+}) => {
+
+    const [columns, setColumns] = React.useState<IColumn[]>([]);
+    const [items, setItems] = React.useState<any[]>([]);
+    const [fileName, setFileName] = React.useState<string>("");
+    const [callerId, setCallerId] = React.useState<string>("");
+
+    const onSubmit = React.useCallback(() => {
+        myDaliService.GetProtectedList({ FileName: fileName, CallerId: callerId }).then(response => {
+            const data = csvParse(response.Result);
+            setColumns(data.columns.map((col, idx) => {
+                return {
+                    key: col,
+                    name: col,
+                    fieldName: col,
+                    minWidth: 100
+                };
+            }));
+            setItems(data);
+        }).catch(err => logger.error(err));
+    }, [fileName, callerId]);
+
+    return <HolyGrail
+        header={<span><TableGroup fields={{
+            "FileName": { label: nlsHPCC.FileName, type: "string", value: fileName },
+            "CallerId": { label: nlsHPCC.CallerID, type: "string", value: callerId },
+        }} onChange={(id, value) => {
+            switch (id) {
+                case "fileName":
```

Comment:
Same as in GetLogicalFilePart.tsx, these should match the casing of the field keys, so FileName and CallerId. Alternatively, if you were wanting to have them be consistent with the variable names, you could change the casing on lines 50 & 51 to be:
`"fileName": { label: ... },`
`"callerId": { label: ... }`
The onChange callback is switching on that "id" to determine what to do, so it just needs to match, either way.